MonMaraicher/Features/Map/FarmerService.swift

```diff
@@ -6,13 +6,18 @@
 //
 
 import Foundation
+import CoreLocation
 
-final class FarmerService {
+protocol FarmerServiceProtocol {
+    func searchFarmers(location: CLLocation) async throws -> Farmers
```

Comment:
a label like `around` or `for` could be nice isn't?
`func searchFarmers(around location: CLLocation)`